firmware/application/bitmap.hpp

```diff
@@ -5607,6 +5607,44 @@ static constexpr Bitmap bitmap_icon_speaker_and_headphones_mute{
     {16, 16},
     bitmap_icon_speaker_and_headphones_mute_data};
 
+static constexpr uint8_t bitmap_icon_shift_data[] = {
+    0x00,
+    0x00,
+    0x80,
+    0x00,
+    0x40,
+    0x01,
+    0x20,
+    0x02,
+    0x10,
+    0x04,
+    0x08,
+    0x08,
+    0x04,
+    0x10,
+    0x3C,
+    0x1E,
+    0x20,
+    0x02,
+    0x20,
+    0x02,
+    0x20,
+    0x02,
+    0x20,
+    0x02,
+    0x20,
+    0x02,
+    0x20,
+    0x02,
+    0xE0,
+    0x03,
+    0x00,
+    0x00,
+};
+static constexpr Bitmap bitmap_icon_shift{
```

Comment:
I'm finding it difficult to tell the difference between the 3 foreground colors of the shift icon.  I would propose to make the arrow solid (filled in), versus an outline.

Reply:
Another possible alternative might be a thicker arrow outline (anything to make the color more visible).

Reply:
I made a more solid icon.